src/Ast/PhpDoc/MethodTagValueGenericNode.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDocParser\Ast\PhpDoc;
+
+use PHPStan\PhpDocParser\Ast\Node;
+use PHPStan\PhpDocParser\Ast\NodeAttributes;
+use PHPStan\PhpDocParser\Ast\Type\TypeNode;
+
+class MethodTagValueGenericNode implements Node
```

Comment:
I feel like this doesn't need a new node, but TemplateTagValueNode can be reused instead. We'd also gain default template types (https://github.com/phpstan/phpdoc-parser/pull/148) automatically :)